client/tests/util/getPage.js

```diff
@@ -38,6 +38,20 @@ const startServer = async serverOrClient => {
             }
         );
 
+        server.on('error', error => {
+            console.info(`Error found in startServer process: ${error}`); // eslint-disable-line no-console
```

Comment:
It seems as though the behavior of the tests will be meaningless from this moment forward, and the test runner will report that misleading error much more prominently than a stray console message. Shouldn't we interrupt test execution, instead?

Reply:
I agree. I've updated the code to exit on error.